### PR TITLE
alphabetised the Where to Start section and included Play with @moby tool

### PR DESCRIPTION
**Pull request by vegasbrianc**
Added [L 93] http://play-with-moby.com/

**PWM is a Moby web based playground which allows you to try different components of the platform in seconds. It gives you the experience of having a free Alpine Linux Virtual Machine in the cloud where you can build and run Moby projects and even create clusters to experiment.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)
Yes
- The project submitted conforms to the quality standards outlined in the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md)
Yes
- Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.
Yes

RULES (because you probably didn't read the guidelines): 
* ORDER: The entries are in lexical order (English letter order) 
* CREDIT: Each entry must contains the name of the author with a link to its github account!
* LINK: the link should be to the source code (https://github/account/project) not the website (https://myproject.io <- no!)
* MONEY: indicates if one needs a paying subscription to use that project you could just add ($$$) next to the name ([my project - $$$] (link to my project) )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,20 +76,21 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
...
+* [Docker Caveats](http://docker-saigon.github.io/post/Docker-Caveats/) What You Should Know About Running Docker In Production (written 11 APRIL 2016) __MUST SEE__
+* Check out [Docker Cheat Sheet][docker-cheat-sheet] by [@wsargent][wsargent] __MUST SEE__
+* [Docker Containers on the Desktop][jessblog] - The **funniest way** to learn about docker by [@jessfraz][jessfraz] who also gave a [presentation][jessvid] about it @ DockerCon 2015 (Tips: checkout her [dotfiles][jessfrazdotfiles] and her [dockerfiles][jessfrazdockerfiles])
+* [Docker Curriculum](https://prakhar.me/docker-curriculum/): A comprehensive tutorial for getting started with Docker. Teaches how to use Docker and deploy dockerized apps on AWS with Elastic Beanstalk and Elastic Container Service.
 * [Docker Documentation](https://docs.docker.com/)
-* [Docker Training](http://training.docker.com/) - Includes a free self-paced hands-on tutorial (free registration required or sign-in with DockerHub ID)
 * Read this complete article: [Basics – Docker, Containers, Hypervisors, CoreOS](http://etherealmind.com/basics-docker-containers-hypervisors-coreos/)
```
Comment (devhkr) [inline README.md]: May I suggest to remove `Read this complete article`, so the link `Basics – Docker, Containers, Hypervisors, CoreOS` got listed alphabetically first? IMO, a good article to start with in this section.

File: README.md [hunk 2]
```diff
@@ -76,20 +76,21 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 _Source:_ [What is Docker](https://www.docker.com/what-docker)
 
 # Where to start?
-* [Play With Docker](http://training.play-with-docker.com/) - A great way to get started with Docker from beginner to advanced users. Docker runs directly in your browser.
+* [Docker Caveats](http://docker-saigon.github.io/post/Docker-Caveats/) What You Should Know About Running Docker In Production (written 11 APRIL 2016) __MUST SEE__
+* Check out [Docker Cheat Sheet][docker-cheat-sheet] by [@wsargent][wsargent] __MUST SEE__
```
Comment (devhkr) [inline README.md]: Just remove the word `Check out`

File: README.md (hunk 1 above)
Comment (devhkr) [inline README.md]: Do you know a good short description, or just leave it?

Comment (vegasbrianc) [inline README.md]: I think it is sufficient, I will remove the sentence you indicated.

**Comment by vegasbrianc** [comment]
@devhkr Updated. Thanks again for the feedback

**Comment by devhkr** [comment]
Thanks @vegasbrianc for reordering the 'Where to start' section and adding Play-With-Moby

**Pull request merged**